Userland/Games/BrickGame/BrickGame.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "BrickGame.h"
+#include "Tetris.h"
+#include <LibConfig/Client.h>
+#include <LibGUI/MessageBox.h>
+#include <LibGUI/Painter.h>
+#include <LibGfx/Bitmap.h>
+#include <LibGfx/Font/FontDatabase.h>
+
+BrickGame::BrickGame(StringView app_name)
+    : m_app_name { app_name }
+    , m_state { GameState::Idle }
+    , m_tetris(make<Tetris>())
+{
+    set_font(Gfx::FontDatabase::default_fixed_width_font().bold_variant());
+    m_high_score = Config::read_i32(m_app_name, m_app_name, "HighScore"sv, 0);
+    reset();
+}
+
+void BrickGame::reset()
+{
+    m_state = GameState::Active;
+    m_tetris->reset();
+    stop_timer();
+    start_timer(15);
+    if (!m_tetris->new_block()) {
+        VERIFY(!"Must never happen");
```

Comment:
This can just be replaced with `VERIFY_NOT_REACHED`
Additionally, single-statement `if` blocks should not have curly braces.